style.css

```diff
@@ -82,354 +75,334 @@ button:disabled {
-  margin-left: 16px;
-  font-family: 'Crete Round', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 32px;
-  line-height: 44px;
+.item > h3,
+.item > p,
+.item > ul {
   color: #fff;
+  margin: 12px;
 }
 
-.projects p {
-  height: 87px;
-  margin-top: 12px;
-  margin-left: 16px;
-  color: white;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
+.item li {
+  background-color: rgba(255, 255, 255, 0.24);
 }
 
-footer ul {
-  justify-content: center;
-  display: flex;
-  align-items: center;
+.item > button {
   width: 100%;
-  align-self: center;
 }
 
-.projects ul {
-  margin-left: 16px;
-  margin-top: 0;
+/* about me section */
+.about-section {
+  background-color: #1c1a19;
+  margin-top: 120px;
 }
 
-.programming-languages {
-  display: flex;
-  list-style: none;
-  padding-left: 0;
+.about-me-section {
+  background-image: url(images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
 }
 
-.programming-languages > * {
+.about-me-section h2 {
+  font-family: "Crete Round", sans-serif;
   font-style: normal;
-  font-weight: 500;
-  background: rgba(202, 194, 194, 0.24);
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
 }
 
-.project-btn {
-  background: #ff6b00;
-  border: none;
-  height: 50px;
-  left: 0;
-  top: 0;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  color: white;
+.about-me-section > p {
+  color: #fff;
 }
 
-.about-me {
-  margin-top: 0;
-  padding-top: 0;
-  height: 700px;
-  background: #1c1a19;
-  background-image: url(images/about.png);
-  background-repeat: no-repeat;
-  background-position: 100% 100%;
-  display: flex;
+.about-me-section button {
+  width: 161px;
+}
+
+.border-about-me-section {
+  border-bottom: 1px solid #fff;
+  margin-top: 201px;
+}
+
+/* programming-languages section */
+.programming-languages {
   flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 120px 0;
 }
 
-.about-me h2 {
-  height: 79px;
-  margin-top: 120px;
-  margin-left: 6px;
-  font-family: 'Crete Round', sans-serif;
+.programming-languages h4 {
+  display: list-item;
+  text-align: center;
+  font-family: "InaiMathi", sans-serif;
   font-style: normal;
-  font-weight: 400;
-  font-size: 70px;
-  line-height: 88px;
-  letter-spacing: 0.35px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: #fff;
+}
+
+.programming-languages h4::marker {
   color: #ff6b00;
 }
 
-.about-me p {
-  font-family: 'Inter', sans-serif;
+.programming-languages li {
+  text-align: left;
+  font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 16px;
-  color: #f4f5f7;
-  padding: 12px 16px;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
 }
```

Comment:
- [ ] As the last code reviewer said 👉🏽 Kindly align the language frameworks and skills like that of Figma. Please 🙏🏽 make sure that both desktop and mobile versions look similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/206717096-fa77ac27-e12a-4be1-9b9f-14231c2cb5b1.png) | ![image](https://user-images.githubusercontent.com/58642949/206718758-41dab930-3741-4037-96ed-31667bfa068d.png) |